ccd-definition/ComplexTypes/CaseToPostLRspec.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "accessCode",
+    "FieldType": "Text",
+    "ElementLabel": "Access Code",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseToPostLRspec",
+    "ListElementCode": "respondentCaseRole",
```

Comment:
So we have citizenCaseRole and also respondentCaseRole.
I think we just need one of them.

Reply:
After discussion, it will be confirm if the role is same or not when doing the citizen UI part